IFC4x3/Properties/c/CableArrangerPosition_1bt9RJJNv7CeH$s$fdSlY0/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the directional position of the cable fitting: vertical, horizontal, front or rear.
```

Comment:
An inquiry here. These position enums are super tricky. Are you sure this can be understood?
```suggestion
Indicates the directional position of the cable fitting: vertical, horizontal, front or rear.
```